components/homme/src/prim_main.F90

```diff
@@ -20,7 +20,7 @@ program prim_main
   use element_mod,      only: element_t
   use common_io_mod,    only: output_dir, infilenames
   use common_movie_mod, only: nextoutputstep
-  use perf_mod,         only: t_initf, t_prf, t_finalizef, t_startf, t_stopf ! _EXTERNAL
+  use perf_mod,         only: t_initf, t_prf, t_finalizef, t_startf, t_stopf, t_disablef, t_enablef ! _EXTERNAL
```

Comment:
`t_enablef` is not used (will generate yet another compiler warning). Let's prune it.